docker-compose.yml

```diff
@@ -252,7 +256,7 @@ services:
 
   zkprover-mock:
     container_name: zkprover-mock
-    image: hermeznetwork/zkprover-mock:latest
```

Comment:
Remember to switch to this image once you have access to dockerhub login

Reply:
good point @KonradIT, done here 2dab38e5f631e394800f1d48c8215d2539d4da8f